### ISSUE TITLE
style swap equals color mixture in some cases?

### ISSUE THREAD

**Issue by John-HW-Cao** [rtqichen/style-swap#20]
With the default parameters, I have got this result.
![test](https://user-images.githubusercontent.com/20965401/27618077-ca6b84aa-5bec-11e7-8a93-8f1fbf4a8c0d.jpeg)

It seems to me that yellow + blue = green. 
But in the paper, the feature map of the content image on each patch is replaced the closest one of the style image. Then I thought that the merged image should take the yellow color.

Could anyone please help me understand this?

**Comment by rtqichen** [comment]
It's due to the addition of overlapping patches during style swap. So apparently yellow+yellow=green inside the VGG network. Yeah, adding activations might not be the best idea..

**Comment by John-HW-Cao** [comment]
Thank you for your quick response.

Yes, it is from optimization. And I will follow your advice and check the effect of the averaging.

**Comment by John-HW-Cao** [comment]
I have found that the reason is that in this case, the 100 iterations of optimization is not enough. 
After running about 800 iterations with learningRate=0.05, I have got 
![content_stylized](https://user-images.githubusercontent.com/20965401/27682918-e23e4882-5cf6-11e7-860a-1451052bf655.png)

The color is good now, but the edges in the stylized image are not satisfying. I think the reason could be that the activation space in relu3_1 is not sensitive to the edges of the images.

I think the color problem is solved now and I close this issue.
